src/core_scan/Core_scan.ml

```diff
@@ -171,6 +171,12 @@ let replace_named_pipe_by_regular_file path =
         Fpath.v tmp_path
     | _ -> path
 
+let add_additional_targets config n =
+  (* Print additional target count so the Python progress bar knows *)
+  match config.output_format with
+  | Json true -> pr @@ string_of_int n
```

Comment:
please do not use @@.

Reply:
What's the rationale for that? In this instance, I think this is simpler/easier to read than using parens or `|>`. In any event, if we want to avoid `@@` it would be nice if we could document that choice and the reason for it somewhere (it isn't currently, to my knowledge).